src/openfl/text/TextField.hx

```diff
@@ -61,6 +61,7 @@ class TextField extends InteractiveObject implements IShaderDrawable {
 	private static var __defaultTextFormat:TextFormat;
 	private static var __missingFontWarning = new Map<String, Bool> ();
 	private static inline var __scrollStep = 24;
+	private static inline var __mouseWheelScrollLines = 3;
```

Comment:
I hard-coded this because Lime seems to send some crazy delta value from the browser (like `-100`), but this affects other `MOUSE_WHEEL` event, so maybe instead we should investigate that and fix the delta.